style.css

```diff
@@ -719,6 +728,7 @@ ul li {
   text-align: center;
   letter-spacing: 0.03em;
   color: rgb(255, 255, 255);
+  cursor: pointer;
 }
 
 .contact-btn:hover {
```

Comment:
- [ ] I think it's better to use the properties for hover provided by the Figma template. It's a project requirement. There are changes in box-shadow and text-align.
- [ ] Same thing for "Get my resume" button.
- [ ] Same thing for "See project" button.